src/Views/batchprocessor.html

```diff
@@ -143,6 +143,10 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
             <div ng-if="vm.cbFlowStats" style="padding-top: 15px;" class="well well-sm">
                 <h3 class="collapsible" ng-click="vm.collapseSection($event, 'flowStatistics')"><i ng-class="!vm.flowStatsCollapsed ? 'fa fa-chevron-right': 'fa fa-chevron-down'"></i>&nbsp;&nbsp;Select Flow Statistics</h3>
                 <div class="collapsible-content" style="background-color: #f5f5f5;">
+                    <!-- Message that no flow statistics are available -->
+                    <div ng-if="vm.flowStatsList.length == 0">
+                        <span>Scenario computation has not been enabled yet for selected region.</span>

```

Comment:
Maybe we get rid of the "yet", which implies that scenarios may be available later? 

```suggestion
                        <span>Scenario computation has not been enabled for selected region.</span>

```

Reply:
I copied the text we are using in the sidebar when scenarios are not available, should we switch that too.

Reply:
Ah didn't realize, let's just stick with this to be consistent!